### ISSUE TITLE
Epi: WooCommerce templates showing by default in Site Editor without Woo ever being activated

### ISSUE THREAD

**Issue by philnick206** [Automattic/themes#8280]
### Quick summary

After activating the Epi theme on a simple site the default Woo templates will be included in the Site Editor like Cart, Checkout, Single Product, etc.

![Image](https://github.com/user-attachments/assets/5e460a98-da56-4038-9f42-10e6a5fedf04)
   

### Steps to reproduce

1. On a simple site, activate Epi theme
2. Go to Appearance > Editor > Templates

### A clear and concise description of what you expected to happen.

No Woo templates should be present

### What actually happened

Woo templates are present

### Impact

All

### Available workarounds?

Yes, easy to implement

### If the above answer is "Yes...", outline the workaround.

Ignore the templates

### Logs or notes

Reported in 8872228-zd-a8c

**Comment by Robertght** [comment]
📌 REPRODUCTION RESULTS
- Tested on Atomic – Replicated


📌 ACTIONS
- Triaged
- I couldn't replicate this with TT4 or Mann.

**Comment by alaczek** [comment]
@philnick206 I went through the customer interaction and I'm not seeing them having an issue with these templates. Can you help me understand why this is a problem that these templates exist in the themes? 

@iamtakashi are the woo templates included in this theme on purpose?

**Comment by philnick206** [comment]
@alaczek You are correct that the user didn't call out these templates specifically, but if I remember correctly, when looking at the site, they had gone in and edited the Product template at some point (who knows why, I think they thought those were the same as pages), and I noticed it was showing an error because Woo wasn't installed. I think that, generally speaking, having any of those Woo templates there when Woo isn't activated looks bad (since they are broken) and can lead to unnecessary confusion. Thus, I opened this bug.

**Comment by iamtakashi** [comment]
> @iamtakashi are the woo templates included in this theme on purpose?

Yeah, this is in an awkward situation now. The base theme, Archivo was built a couple of years ago when there was nothing coming out from Woo regarding templates/themes. I remember that Woo didn't provide templates, and having those templates was the only way to support Woo in working in a less broken way.

I'm more than happy to remove all the templates from the related themes, Archivo, Epi, Kaze, and Pomme, but I'm not sure what would happen to the sites using Woo. Will removing those templates now break those sites? Can anyone from Woo chime in?
